andromeda/requirements.txt

```diff
@@ -0,0 +1,8 @@
+ipywidgets==7.6.6
+matplotlib==3.7.1
+opencv-python==4.7.0.72
+pandas==2.0.0
+scikit-image==0.20.0
+scikit-learn==1.2.2
+
+
```

Comment:
Now that we have slimmed down this PR I think this requirements file is now out of date.
Looking at the code I think we only using `pandas==2.0.0` and `scikit-learn==1.2.2`.
Could you simplify the requirements.txt file? Thanks.

Reply:
Yes, we do. It's been updated.

Reply:
Thanks.